charts/brig/templates/deployment.yaml

```diff
@@ -66,6 +66,24 @@ spec:
           # TODO: Is this the best way to do this?
           - name: AWS_REGION
             value: "{{ .Values.config.aws.region }}"
+          {{- if .Values.config.proxy.httpProxy }}
+          - name: http_proxy
+            value: {{ .Values.config.proxy.httpProxy | quote }}
+          - name: HTTP_PROXY
+            value: {{ .Values.config.proxy.httpProxy | quote }}
+          {{- end }}
+          {{- if .Values.config.proxy.httpsProxy }}
+          - name: https_proxy
+            value: {{ .Values.config.proxy.httpsProxy | quote }}
+          - name: HTTPS_PROXY
+            value: {{ .Values.config.proxy.httpsProxy | quote }}
+          {{- end }}
+          {{- if .Values.config.proxy.noProxy }}
+          - name: no_proxy
+            value: {{ .Values.config.proxy.noProxy | quote }}
+          - name: NO_PROXY
+            value: {{ .Values.config.proxy.noProxy | quote }}
+          {{- end }}      
```

Comment:
Thank you for all your effort! Some minor nitpicking, but free to integrate it or not :)

In this case, setting the scope can simplify the overall code a bit:
```
{{- with .Values.config.proxy }}
{{- if .httpProxy }}
- name: http_proxy
  value: {{ .httpProxy | quote }}
- name: HTTP_PROXY
  value: {{ .httpProxy | quote }}
{{- end }}
{{- if .httpsProxy }}
- name: https_proxy
  value: {{ .httpsProxy | quote }}
- name: HTTPS_PROXY
  value: {{ .httpsProxy | quote }}
{{- end }}
{{- if .noProxyList }}
- name: no_proxy
  value: {{ join "," .noProxyList | quote }}
- name: NO_PROXY
  value: {{ join "," .noProxyList | quote }}
{{- end }}
{{- end }}
```